crates/subspace-runtime/src/lib.rs

```diff
@@ -585,6 +584,9 @@ parameter_types! {
 // Minimum operator stake must be >= minimum nominator stake since operator is also a nominator.
 const_assert!(MinOperatorStake::get() >= MinNominatorStake::get());
 
+// Stake Withdrawal locking period must be >= Block tree pruning depth
+const_assert!(StakeWithdrawalLockingPeriod::get() >= BlockTreePruningDepth::get());
```

Comment:
Technically, this assert is not needed, because, as per the `do_withdraw_stake` [code](https://github.com/subspace/subspace/blob/deeec211e11b75d1280e95ceb500722c9e892291/crates/pallet-domains/src/staking.rs#L690) they are added.